server/config.ts

```diff
@@ -80,6 +81,7 @@ export default {
         deadline: 10000,
       },
       agent: new AgentConfig(10000),
+      serviceName: get('APPROVED_PREMISES_API_SERVICE_NAME', 'approved-premises', requiredInProduction),
```

Comment:
Does this set a default of `approved-premises` if the environment variable is undefined? If so, line 34 might not ever be undefined in which case we could remove the optionality.

Reply:
There's multiple APIs - approved-premises, hmppsAuth, and tokenVerification, and we only define `serviceName` for approved-premises, hence it being optional